lib/src/base-styles/_nx-counter.scss

```diff
@@ -22,7 +22,7 @@
   min-width: 8px;
 
   &.nx-counter--active {
-    background-color: var(--nx-swatch-blue-10);
+    background-color: var(--nx-swatch-grey-05);
```

Comment:
> I'm not 100% sure if this change is extraneous to this ticket, but I also don't see a ticket to sync NxCounter 🤔
> 
> Also, this will break other component's visual tests, (I think some of our interaction state tickets already does that) and I guess we probably should run the visual test again as a whole once all the interaction tickets are completed in the feature branch

Re: visual tests, I definitely think we will need to do a final sweep of snapshots come the end. I know there are lots of snapshots in containers (ie. modals & tiles) that will now fail with the interaction states, but weren't updated in that ticket (either forgotten or multiple components needing their interaction states updated first). 

As for the NxCounter colour change, I'm open to either dealing with it here or creating a separate bug ticket to fix the colours. I realize that the border colour in the non-active state is also grey-05 and not blue-10 which will need to be updated. Perhaps then a bug ticket? Let me know your thoughts on that one

Reply:
> > I'm not 100% sure if this change is extraneous to this ticket, but I also don't see a ticket to sync NxCounter 🤔
> > Also, this will break other component's visual tests, (I think some of our interaction state tickets already does that) and I guess we probably should run the visual test again as a whole once all the interaction tickets are completed in the feature branch
> 
> Re: visual tests, I definitely think we will need to do a final sweep of snapshots come the end. I know there are lots of snapshots in containers (ie. modals & tiles) that will now fail with the interaction states, but weren't updated in that ticket (either forgotten or multiple components needing their interaction states updated first).
> 
> As for the NxCounter colour change, I'm open to either dealing with it here or creating a separate bug ticket to fix the colours. I realize that the border colour in the non-active state is also grey-05 and not blue-10 which will need to be updated. Perhaps then a bug ticket? Let me know your thoughts on that one

I think we can create a new ticket (doesn't have to be a bug ticket), it can just be like the other interaction states ticket but for nx-counter.

Reply:
Ah sorry, I miss-read the colour miss-matches we're related to the interaction states, definitely a bug ticket would make sense.